py3status/modules/xrandr.py

```diff
@@ -141,17 +156,29 @@ def _set_available_combinations(self):
         available_combinations = set()
         combinations_map = {}
 
+        if self.output_combinations is None:
+            whitelist = None
+        else:
+            whitelist = self.output_combinations.split('|')
+
         self.max_width = 0
-        for output in range(len(self.layout['connected']) + 1):
+        for output in range(1, len(self.layout['connected']) + 1):
             for comb in combinations(self.layout['connected'], output):
```

Comment:
you could remove the `1`s from the line above (165) and then use `output + 1` here